cookiecutter/compat.py

```diff
@@ -131,4 +131,10 @@ def _access_check(fn, mode):
                         return name
         return None
 
+
+def isstr(obj):
+    """Determine if an object is a string."""
+    return isinstance(obj, str if PY3 else basestring)  # NOQA
```

Comment:
With `flake8` installed in a Python 3 virtualenv:

`cookiecutter/compat.py:137:44: F821 undefined name 'basestring'`